cmd/opm/alpha/veneer/cmd.go

```diff
@@ -16,3 +19,18 @@ func NewCmd() *cobra.Command {
 
 	return runCmd
 }
+
+func OpenFileOrReadStdin(cmd *cobra.Command, args []string) (io.ReadCloser, string, error) {
+	var err error = nil
+	var source string = ""
+	var reader io.ReadCloser
+
+	if len(args) == 0 || args[0] == "-" {
+		reader = io.NopCloser(cmd.InOrStdin())
+		source = "stdin"
+	} else {
+		reader, err = os.Open(args[0])
+		source = args[0]
+	}
+	return reader, source, err
```

Comment:
Nit: might be able to shorten this a bit, but take it or leave it depending on your feel for readability.

```suggestion
	if len(args) == 0 || args[0] == "-" {
		return io.NopCloser(cmd.InOrStdin()), "stdin", nil
	}
	file, err := os.Open(args[0])
	return file, args[0], err
```